index.php

```diff
@@ -325,9 +325,28 @@
             $OUTPUT->help_icon('restartstop', 'tool_moodlebox'), 'moodleboxrestartstopsection');
     echo $OUTPUT->box_start('generalbox');
 
+    $restartstopfilename = '.restartstopsettings';
```

Comment:
This would be better done via an admin setting, see 
https://docs.moodle.org/dev/Admin_settings#Individual_settings